docs/developing.md

```diff
@@ -89,3 +89,225 @@ against the running RGW. E.g., `s3cmd mb s3://foo`, to create a new bucket.
 [2]: https://github.com/aquarist-labs/ceph/tree/s3gw
 [3]: https://docs.ceph.com/en/pacific/install/build-ceph/#id1
 [4]: https://github.com/s3tools/s3cmd
+
+## How to build your own containers
```

Comment:
This section (line 93-313) is a duplication of the `build-instructions.md`. Would it make sense to link to this file here?

Reply:
The idea is to concentrate both instructions in this document, which sorts out the UX issue. We could have potentially just added links to the files instead but I wanted to save the user that extra click. I just deleted `build-ui.md`